src/node_modules.cc

```diff
@@ -378,60 +378,56 @@ void BindingData::GetPackageScopeConfig(
 
   Realm* realm = Realm::GetCurrent(args);
   Utf8Value resolved(realm->isolate(), args[0]);
-  auto package_json_url_base = ada::parse(resolved.ToStringView());
-  if (!package_json_url_base) {
+
+  auto current_path_url = ada::parse(resolved.ToStringView());
+  if (!current_path_url) {
     url::ThrowInvalidURL(realm->env(), resolved.ToStringView(), std::nullopt);
     return;
   }
-  auto package_json_url =
-      ada::parse("./package.json", &package_json_url_base.value());
-  if (!package_json_url) {
-    url::ThrowInvalidURL(realm->env(), "./package.json", resolved.ToString());
-    return;
-  }
 
-  std::string_view node_modules_package_path = "/node_modules/package.json";
+  auto current_path_opt = url::FileURLToPath(realm->env(), *current_path_url);
+  CHECK(current_path_opt);
+  // BufferValue path_buffer(realm->isolate(), current_path_opt.value());
+  // // Check if the path has a trailing slash. If so, add it after
+  // // ToNamespacedPath() as it will be deleted by ToNamespacedPath()
+  // bool endsWithSlash = path_buffer.ToStringView().ends_with(
+  //     std::filesystem::path::preferred_separator);
+  // ToNamespacedPath(realm->env(), &path_buffer);
+  // std::string path_value_str = path_buffer.ToString();
+  // if (endsWithSlash) {
+  //   path_value_str.push_back(std::filesystem::path::preferred_separator);
+  // }
```

Comment:
I didn't understand the purpose of this comments. Did you forget them?

```suggestion
```

Reply:
ToNamespacedPath needs to be called here, as current_path_opt is the non-UNC path and won't work if it's longer than 260 characters, but that function requires a BufferValue which requires a v8 value
I was planning to change ToNamespacedPath to accept a string, as it suggests [here](https://github.com/nodejs/node/blob/bc5ebe44f95755e97e60bcab50ea59358d1faee5/src/node_file.cc#L3343)